src/commands/deploy/deploy.ts

```diff
@@ -382,7 +382,7 @@ const uploadDeployBlobs = async ({
     packagePath,
     deployId,
     siteId,
-    token,
+    token: token === null ? undefined : token,
```

Comment:
`token` here is of type `string | null`, while `runCoreSteps` expect it to be `string | undefined`

Overall at this stage there should be a token always, otherwise we would bail much earlier, but stuff like this is annoying with TS and probably would need more refactoring to not have to deal with `null / undefined` cases.